app/src/main/java/org/hisp/dhis/android/dataentry/form/section/viewmodels/FormItemViewModelFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.hisp.dhis.android.dataentry.form.section.viewmodels;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.hisp.dhis.android.core.common.ValueType;
+
+public interface FormItemViewModelFactory {
```

Comment:
Isn't it better to have factory methods in each view model instead of one god class?